src/importMarkdownToLexical.ts

```diff
@@ -79,7 +79,7 @@ function isParent(node: unknown): node is Mdast.Parent {
  * @internal
  */
 export interface MdastTreeImportOptions {
-  root: LexicalRootNode
+  root: LexicalNode
```

Comment:
`LexicalRootNode` is a child class of `LexicalNode`, it's OK to use `LexicalNode` here as we need to insert the new markdown value into a LexicalNode.